src/book/core.clj

```diff
@@ -0,0 +1,45 @@
+(ns book.core)
+(ns book.core
+  (:require [clojure.test :refer [is run-tests with-test]])
```

Comment:
The convention is to close parenthesis on the line above

Reply:
I went through all the code to eliminate brackets on lines where there is nothing but brackets